### PR TITLE
Feature/support for ios14

### LINKED ISSUE #599: [iOS] Add support for iOS 14.
### Description

Check Magic Script Components on iOS 14.

### PR DESCRIPTION
**Pull request by pleszkiewicz**
Fixes #599 

text_node_ios14

### CODE REVIEW

File: ios/RNMagicScriptHostApplication/sources/ViewController.swift [hunk 1]
```diff
@@ -34,9 +34,10 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         setupScene()
         
-        //        setupPrismWithModels()
-        //        setupPrismForHitTest()
-        setupPrismForDialog()
+//        setupPrismWithModels()
+//        setupPrismForHitTest()
+//        setupPrismForDialog()
```
Comment (kpiascik) [inline ios/RNMagicScriptHostApplication/sources/ViewController.swift:39]: why are you keeping commented out code?

File: ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift [hunk 2]
```diff
@@ -246,8 +246,8 @@ class UiTextNodeSpec: QuickSpec {
 
             context("when wrap disabled") {
                 let referenceBoundsSize = CGSize.zero
-                let shortTextRefereneceSizeForBounds = CGSize(width: 0.0426, height: 0.0144)
-                let veryLongTextRefereneceSizeForBounds = CGSize(width: 0.4986, height: 0.0144)
+                let shortTextRefereneceSizeForBounds: CGSize = is_iOS14() ? CGSize(width: 0.0444, height: 0.0168) : CGSize(width: 0.0426, height: 0.0144)
```
Comment (kpiascik) [inline ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift:249]: is this only because we're using a different font for iOS 14?

**Review by kpiascik — APPROVED** [review]
Looks good, please address the comments I have before merging.

File: ios/RNMagicScriptHostApplication/sources/ViewController.swift (hunk 1 above)
Comment (pleszkiewicz) [reply ios/RNMagicScriptHostApplication/sources/ViewController.swift:39]: This project/code is for debug/testing purposes (for developers). It does not affect anything related to magic-script. It just builds sample scenes from swift code (not from JSX). It is also used for Unit tests project (but with an empty scene).

File: ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift (hunk 2 above)
Comment (pleszkiewicz) [reply ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift:249]: Yes, do you see any better way of testing font/text size?

Comment (kpiascik) [reply ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift:249]: No, this is fine for now and it being linked to the same commit where we change the iOS 14 Font then it makes sense
